plugins/omar-geoscript-plugin/grails-app/services/omar/geoscript/ExportService.groovy

```diff
@@ -1,30 +1,50 @@
 package omar.geoscript
 
+import geoscript.feature.Field
 import geoscript.filter.Filter
 import geoscript.layer.Layer
 import geoscript.workspace.Directory
 import geoscript.workspace.Workspace
+import grails.core.GrailsApplication
 import org.geotools.data.DataStoreFinder
+
 import java.nio.file.Files
 import geoscript.GeoScript
 
 class ExportService {
-  
+
+    GrailsApplication grailsApplication
+
     def exportShapeZip(String typeName, String filter, Integer maxFeatures, Integer startIndex) {
 
         File directory = Files.createTempDirectory("shape-").toFile()
         Directory shapeDir = new Directory(directory)
+        Layer destLayer
 
         def (prefix, layerName) = typeName?typeName.split(':'):[null,null]
+
         Layer srcLayer  = findLayer(prefix, layerName)
-        Layer destLayer = shapeDir.create(srcLayer.schema)
+
+        List<Field> excludeFields =  grailsApplication.config.getProperty('geoscript.export.excludes', List).collect { srcLayer.schema.field(it) }
+
+        if(excludeFields) {
+            def srcSchemaUpdated = srcLayer.schema.removeFields(excludeFields, "raster_entry")
```

Comment:
wish we didn't have to hard code raster_entry but I guess I don't foresee us providing other things to export at the moment.